frontend/packages/schema-editor/src/components/SchemaInspector/EnumField.tsx

```diff
@@ -1,62 +1,71 @@
 import type { KeyboardEvent } from 'react';
 import React, { useEffect, useState } from 'react';
 import { IconButton } from '../common/IconButton';
-import { LegacyTextField } from '@digdir/design-system-react';
+import { Textfield } from '@digdir/design-system-react';
 import classes from './EnumField.module.css';
-import { getDomFriendlyID } from '../../utils/ui-schema-utils';
 import { IconImage } from '../common/Icon';
 import { useTranslation } from 'react-i18next';
 
-export interface IEnumFieldProps {
+export type EnumFieldProps = {
   path: string;
   value: string;
   readOnly?: boolean;
-  fullWidth?: boolean;
   isValid?: boolean;
   onChange: (value: string, oldValue?: string) => void;
   onDelete?: (path: string, key: string) => void;
   onEnterKeyPress?: () => void;
-}
+  baseId: string;
+};
 
-export const EnumField = (props: IEnumFieldProps) => {
-  const [val, setVal] = useState(props.value);
+export const EnumField = ({
+  path,
+  value,
+  readOnly,
+  isValid,
+  onChange,
+  onDelete,
+  onEnterKeyPress,
+  baseId,
+}: EnumFieldProps) => {
+  const [val, setVal] = useState(value);
```

Comment:
Should we clean up the naming as we go too? Looks like this useState has been named val/setVal just because value is used within the props. Let's make it more clear than it is. 🤩 

```suggestion
  const [inputValue, setInputValue] = useState(value);
```